gravitee-apim-portal-webui/src/app/pipes/markdown-description.pipe.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Pipe, PipeTransform, SecurityContext } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+import { Api } from '../../../projects/portal-webclient-sdk/src/lib';
+import { MarkdownService } from '../services/markdown.service';
+import { ConfigurationService } from '../services/configuration.service';
+
+@Pipe({
+  name: 'markdownDescription',
+})
+export class MarkdownDescriptionPipe implements PipeTransform {
+  constructor(private configService: ConfigurationService, private markdown: MarkdownService, private sanitizer: DomSanitizer) {}
+
+  transform(api: Api): Api {
+    return api
+      ? {
+          ...api,
+          description: this.renderDescription(api.description),
```

Comment:
If the description is in markdown, then there will be auto-generated links to other pages. One of the new syntaxes ( [here](https://github.com/gravitee-io/gravitee-api-management/pull/6552) ) needs to have the list of pages of a given API to calculate any links that use the syntax. 

This case needs to be handled: either we fetch the pages for each API shown (@anthonyjgatti said that this is fine even if performance is affected) or find another solution.

Reply:
As discussed, links are disabled at the moment to avoid a perf issue

<img width="1066" alt="Screenshot 2024-02-29 at 10 44 50" src="https://github.com/gravitee-io/gravitee-api-management/assets/3619261/96a12667-3895-413a-8aa8-39600ec79f12">
<img width="478" alt="Screenshot 2024-02-29 at 10 44 35" src="https://github.com/gravitee-io/gravitee-api-management/assets/3619261/60a0b834-6d8d-4bce-8205-1b8abbc058bf">